### PR TITLE
adds the clowns pocket to brain damage lines

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball
experimental: Brain damage lines now include stalking.
/:cl:

### CODE REVIEW

**Comment by Iamgoofball** [comment]
what the fuck is with all the red highlighting

**Comment by Durkel** [comment]
Wew, lad. What did you do?

File: strings/traumas.json [hunk 1]
```diff
@@ -1,68 +1,68 @@
...
-        "@pick(semicolon)wtf??????????? @pick(bug)",
-        "@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)",
-        "DON'T EVER TUCH ME"
+        "@pick(semicolon)IM A PONY NEEEEEEIIIIIIIIIGH @pick(ramen)",
+        "without oxigen blob don't evoluate? @pick(ramen)",
+        "@pick(semicolon)CAPTAINS A COMDOM @pick(ramen)",
+        "@pick(semicolon)@pick(faggot_traitor) @pick(george) @pick(mellens) is grifing me HALP!!! @pick(ramen)",
+        "can u give me @pick(mutations)? @pick(ramen)",
+        "THe saiyans screwed @pick(ramen)",
+        "Bi is THE BEST OF BOTH WORLDS> @pick(ramen),
```
Comment (pigeonsk) [inline strings/traumas.json]: \"

Comment (Iamgoofball) [reply strings/traumas.json]: Fuck

**Comment by Iamgoofball** [comment]
looks like ramen posting is not meant to be

File: strings/traumas.json [hunk 2]
```diff
@@ -144,6 +144,19 @@
         ".h"
     ],
 
+	"ramen": [
+        "",
```
Comment (iskyp) [inline strings/traumas.json]: is there a space here before the quotes?

**Pull request closed without merging**